index.html

```diff
@@ -3515,10 +3515,10 @@ <h5>JWT Encoding</h5>
             </p>
 
             <p>
-If a JWS is present, the digital signature either refers to the <a>issuer</a>
+If a JWS is present, the digital signature refers to either the <a>issuer</a>
```

Comment:
```suggestion
If a JWS is present, the digital signature refers either to the <a>issuer</a>
```
Makes the construction `either to ... or to ...`, replacing the incorrect `to either ... or to ...`

Reply:
Happy to accept this but when I select Commit suggestion and commit changes it says "invalid email for web commit. Refresh and try again" but if I do refresh it just goes around in the same circle again.